drakrun/drakrun/draksetup.py

```diff
@@ -41,14 +54,33 @@ def install(storage_backend, disk_size, iso_path, zfs_tank_name, max_vms, unatte
     os.makedirs(os.path.join(LIB_DIR, "profiles"), exist_ok=True)
     os.makedirs(os.path.join(LIB_DIR, "volumes"), exist_ok=True)
 
+    conf = configparser.ConfigParser()
+    conf.read(os.path.join(ETC_DIR, "config.ini"))
+    conf_patched = False
+
+    out_interface = conf.get('drakrun', 'out_interface').strip()
+    
+    if not out_interface:
+        default_if = find_default_interface()
+        logging.info(f"Detected default network interface: {default_if}")
+        conf['drakrun']['out_interface'] = default_if
+        conf_patched = True
```

Comment:
somebody may not want to allow guest Internet access at all, but this code would override intentionally-empty setting

perhaps some `net_enable` bool switch should be also added